tests/waku_store_legacy/test_resume.nim

```diff
@@ -1,332 +1,340 @@
 {.used.}
 
-import
-  std/[options, tables, sets],
-  testutils/unittests,
-  chronos,
-  chronicles,
-  libp2p/crypto/crypto
-import
-  ../../waku/common/databases/db_sqlite,
-  ../../waku/waku_archive/driver,
-  ../../waku/waku_archive/driver/sqlite_driver/sqlite_driver,
-  ../../waku/node/peer_manager,
-  ../../waku/waku_core,
-  ../../waku/waku_core/message/digest,
-  ../../waku/waku_store_legacy,
-  ../waku_store_legacy/store_utils,
-  ../waku_archive/archive_utils,
-  ./testlib/common,
-  ./testlib/switch
-
-procSuite "Waku Store - resume store":
-  ## Fixtures
-  let storeA = block:
-    let store = newTestMessageStore()
-    let msgList =
-      @[
-        fakeWakuMessage(
-          payload = @[byte 0], contentTopic = ContentTopic("2"), ts = ts(0)
-        ),
-        fakeWakuMessage(
-          payload = @[byte 1], contentTopic = ContentTopic("1"), ts = ts(1)
-        ),
-        fakeWakuMessage(
-          payload = @[byte 2], contentTopic = ContentTopic("2"), ts = ts(2)
-        ),
-        fakeWakuMessage(
-          payload = @[byte 3], contentTopic = ContentTopic("1"), ts = ts(3)
-        ),
-        fakeWakuMessage(
-          payload = @[byte 4], contentTopic = ContentTopic("2"), ts = ts(4)
-        ),
-        fakeWakuMessage(
-          payload = @[byte 5], contentTopic = ContentTopic("1"), ts = ts(5)
-        ),
-        fakeWakuMessage(
-          payload = @[byte 6], contentTopic = ContentTopic("2"), ts = ts(6)
-        ),
-        fakeWakuMessage(
-          payload = @[byte 7], contentTopic = ContentTopic("1"), ts = ts(7)
-        ),
-        fakeWakuMessage(
-          payload = @[byte 8], contentTopic = ContentTopic("2"), ts = ts(8)
-        ),
-        fakeWakuMessage(
-          payload = @[byte 9], contentTopic = ContentTopic("1"), ts = ts(9)
-        ),
-      ]
-
-    for msg in msgList:
-      require store
-      .put(
-        DefaultPubsubTopic,
-        msg,
-        computeDigest(msg),
-        computeMessageHash(DefaultPubsubTopic, msg),
-        msg.timestamp,
+when defined(waku_exp_store_resume):
```

Comment:
Ah, I just applied the same compile flag here, due it was excluded only on all_test_waku level but not in waku_store_legacy/test_all.nim... it caused me trouble when was running specific protocol tests instead of all_test_waku.
So github compara a bit stupid on this as only when(...): conditional compile was added, all other are just indentation change.